tests/util/checker/linter/README.md

```diff
@@ -0,0 +1,83 @@
+# golinter
+
+golinter applies customized lint rules to our test files. Test files are categorized into
+unit tests, integration tests, and end to end tests based on file locations and file names, and
+different rules are applied accordingly.
+
+golinter is baed on [Checker](../README.md), and this package provides the custom [Rule](../rule.go)s
+implementation.
+
+
+## End To End Tests
+
+All "_test.go" files in a "e2e" directory hierarchy are considered as end to end tests.
+
+Example: 
+```bash
+/istio/tests/e2e/tests/simple/simple_tests.go
+```
+
+#### Rules
+
+1. TBD
+
+
+## Integration Tests
+
+All "_test.go" files in a "integration" directory hierarchy, or with "_integ_test.go" suffix are
```

Comment:
in a "integ" directory hierarchy.

Reply:
If you want to keep "integration" directory hierarchy, then please update GetRules() in rules_matcher.go.